cfgov/jobmanager/models.py

```diff
@@ -102,8 +103,8 @@ def get_absolute_url(self):
 
     def save(self):
         if self.date_created == None:
```

Comment:
@vickumar1981 I think some context here, too is worth noting: this is a piece of larger work @chosak is doing to move the jobmanager models to Wagtail entirely.